todo_app/tests/UITests/test_selenium.py

```diff
@@ -0,0 +1,99 @@
+from ast import Delete
+import os
+from threading import Thread
+from tkinter import Variable
+from todo_app import app
+import pytest
+import requests
+from selenium import webdriver
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # create the new board & update the board id environment variable
+    board_id = create_trello_board()
```

Comment:
Your structure here looks good but there are a few issues you won't find until you start actually trying to run a test using this, e.g. this function expects a board name, and relies on environment variables being set that aren't necessarily yet - e.g. here we'll want to load in `.env` like you did `.env.test` in the integration test

Reply:
added changes as suggested .
as discussed will merge and extend for module 7 - will come back to selenium once caught up